web/src/main/java/io/fabric8/launcher/web/endpoints/BoosterCatalogEndpoint.java

```diff
@@ -53,11 +53,14 @@ public Response getMissions() {
             JsonArrayBuilder runtimes = createArrayBuilder();
             JsonObjectBuilder mission = createObjectBuilder()
                     .add("id", m.getId())
-                    .add("name", m.getName())
-                    .add("suggested", m.isSuggested());
+                    .add("name", m.getName());
             if (m.getDescription() != null) {
                 mission.add("description", m.getDescription());
             }
+            if (m.getMetadata() != null && !m.getMetadata().isEmpty()) {
+                mission.add("metadata", mapToJson(m.getMetadata()));
```

Comment:
Can we make it part of the same object? I think adding `metadata` here would confuse the consumer

Reply:
Confuse? How?

And I don't like the idea of metadata elements possibly overwriting elements in the parent object. That's something that would not be clear at all to the person adding the metadata in the .yaml file.

Reply:
That is a good point, in that case we need to figure out how to describe that in the Swagger document.

Reply:
Perhaps a type of `object` and an explanation that it is a copy of what is found inside the booster catalog itself?

Reply:
WFM